app/main.py

```diff
@@ -1 +1,19 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    parsed_command = command.split()
+    path = parsed_command[-1].split("/")
+    directory = ""
+
+    for i in range(len(path) - 1):
+        directory += path[i]
+        if not os.path.exists(directory):
+            os.mkdir(directory)
+        directory += "/"
+    directory += path[-1]
+
+    with open(parsed_command[1], "r") as f:
```

Comment:
`f` and `c` are not very obvious names for file handles. also you can use tools from `shutil` lib or `pathlib` to move files